docs/README.md

```diff
@@ -61,6 +61,7 @@ Metabase's reference documentation.
 #### Query builder
 
 - [Asking questions](./questions/query-builder/introduction.md)
+- [Visualizing data](./questions/sharing/visualizing-results.md)
```

Comment:
Promoting Visualizations up from Sharing since people tend to care about charts a lot.